### PR TITLE
Polish Dynatrace changes

### PR DESCRIPTION
**Pull request by izeye**
This PR polishes Dynatrace changes a bit.

### CODE REVIEW

File: implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java [hunk 1]
```diff
@@ -327,6 +326,11 @@ private void send(List<String> metricLines) {
         }
     }
 
+    private String getTruncatedBody(HttpSender.Response response) {
+        return StringUtils.truncate(response.body(), LOG_RESPONSE_BODY_TRUNCATION_LIMIT,
+                LOG_RESPONSE_BODY_TRUNCATION_INDICATOR);
```
Comment (arminru) [inline implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java:331]: Now the constants are only used in one single place so you could consider using the values directly instead.

**Review by arminru — APPROVED** [review]

File: implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java (hunk 1 above)
Comment (izeye) [reply implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java]: @arminru Thanks for the suggestion! I applied it.

**Review by arminru — APPROVED** [review]

File: implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java (hunk 1 above)
Comment (arminru) [reply implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java]: Sure, thank you! :)

**Review by shakuzen — APPROVED** [review]
Thanks. The related Dynatrace changes were backported to 1.9 recently, so could you rebase this pull request onto 1.9.x, please?

**Comment by izeye** [comment]
@shakuzen Sure, I changed the base branch to 1.9.x.

**Pull request merged**